drivers/usb/host/xhci-ipq.c

```diff
@@ -57,10 +59,13 @@ void ipq_reset_usb_phy(void *data)
 		return;
 	}
 
-	if(platdata->rst_gpio != -1) {
-		gpio_direction_output(platdata->rst_gpio, GPIO_OUT_LOW);
-		mdelay(1000);
-		gpio_direction_output(platdata->rst_gpio, GPIO_OUT_HIGH);
+	if (platdata->rst_gpio_count > 0) {
```

Comment:
This is really nitpicking. If you set `rst_gpio_count` to zero in the failure case below, you can remove this condition.

Reply:
Done.